lib/hammer_cli_foreman_remote_execution/job_invocation.rb

```diff
@@ -39,6 +39,7 @@ class InfoCommand < HammerCLIForeman::InfoCommand
       extend WithoutNameOption
       include BaseOutput
       option '--show-inputs', :flag, _('Show the complete input of the job')
+      option '--show-host-status', :flag, _('Show job status for the hosts.')
```

Comment:
Usually option descriptions don't end with a dot, unless there are multiple sentences in the description